movement/io/load_poses.py

```diff
@@ -134,6 +134,16 @@ def from_file(
         ``movement`` dataset containing the pose tracks, confidence scores,
         and associated metadata.
 
+
+    Notes
+    -----
+    In movement, pose data can only be loaded if all individuals have
+    the same set of keypoints (i.e., the same labeled body parts).
+    While DeepLabCut supports assigning different keypoints to different
+    individuals (see the `DeepLabCut documentation for multi-animal project,
+    <https://deeplabcut.github.io/DeepLabCut/docs/maDLC_UserGuide.html#b-configure-the-project>`_),
+    this feature is not currently supported in movement.
```

Comment:
I actually think we can remove this note from the docstring of the `from_file()` function. We already have it in two places and I don't want to have too much duplication. I'm aware it was my suggestion to also add it here but I think I changed my mind. Users can always refer to the docs of the `from_dlc_file()` function by using the "See Also" section of the docstring below.